nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/main/java/org/apache/nifi/processors/attributes/UpdateAttribute.java

```diff
@@ -143,9 +144,10 @@ public ValidationResult validate(String subject, String input, ValidationContext
         }
     };
 
+    public static final String DELETE_ATTRIBUTES_EXPRESSION_NAME = "Delete Attributes Expression";
     // static properties
     public static final PropertyDescriptor DELETE_ATTRIBUTES = new PropertyDescriptor.Builder()
-            .name("Delete Attributes Expression")
+            .name(DELETE_ATTRIBUTES_EXPRESSION_NAME)
             .description("Regular expression for attributes to be deleted from FlowFiles.")
```

Comment:
@JPercivall my goal was to preserve existing behavior, I was looking at what the code does, not the documentation.  I'll try to clarify both here and in the doc.